src/sdk-installer.ts

```diff
@@ -19,12 +19,6 @@ export async function installAndroidSdk(apiLevel: string, target: string, arch:
     const isOnMac = process.platform === 'darwin';
     const isArm = process.arch === 'arm64';
 
-    if (!isOnMac) {
-      await exec.exec(`sh -c \\"sudo chown $USER:$USER ${process.env.ANDROID_HOME}/platform-tools -R`);
-      await exec.exec(`sh -c \\"sudo chown $USER:$USER ${process.env.ANDROID_HOME}/cmdline-tools/latest -R`);
-      await exec.exec(`sh -c \\"sudo chown $USER:$USER ${process.env.ANDROID_HOME}/build-tools/${BUILD_TOOLS_VERSION} -R`);
-    }
```

Comment:
Originally we needed to change the owner to accept preview SDK license: https://github.com/ReactiveCircus/android-emulator-runner/commit/49a3d042e9cc6a9f969d767b7e63b59d85a5cb83#diff-b14a072bf3d3c4d7027a549c57f0c1880f60206c24fc202ce4471ab9c0a5881cR23-R28

But this seems no longer required.